cluster/shard_mapper.go

```diff
@@ -154,10 +150,15 @@ func (r *RemoteMapper) Open() (err error) {
 
 	// Decode the first response to get the TagSets.
 	r.tagsets = r.bufferedResponse.TagSets()
+	r.fields = r.bufferedResponse.Fields()
```

Comment:
`RemoteMapper` wasn't passing fields from the remote shard back.  This was causing a panic.